src/utils/searchHelpers.ts

```diff
@@ -200,3 +201,9 @@ export const syncParamsWithSearchFields = (params: URLSearchParams) => {
 
   return params;
 };
+
+export const deleteFromParameter = (params: URLSearchParams, navigate: NavigateFunction) => {
+  const syncedParams = syncParamsWithSearchFields(params);
+  syncedParams.delete(SearchParam.From);
```

Comment:
Siden denne endrer på ticket søk burde vi sikre at den fjerner query param som brukes der også
```suggestion
  syncedParams.delete(TicketSearchParam.From);
```

Reply:
Om dette er noe som skal brukes for andre ting enn ticket søk burde vi ha et eget argument som sier hva som er pagineringsparam til det gitte søket.
`param: SearchParam.From|TicketSearchParam.From|ResultParam.From|(osv...)`

Men bare å vente med det til det blir behov